django-stubs/contrib/admin/options.pyi

```diff
@@ -154,7 +156,7 @@ class ModelAdmin(BaseModelAdmin[_ModelT]):
     delete_selected_confirmation_template: _TemplateForResponseT | None
     object_history_template: _TemplateForResponseT | None
     popup_response_template: _TemplateForResponseT | None
-    actions: Sequence[Callable[[ModelAdmin, HttpRequest, QuerySet], None] | str] | None
+    actions: Sequence[_ActionCallable[Any, _ModelT] | str] | None
```

Comment:
It s unfortunate that we have replace`ModelAdmin` with `Any` here. But hopefully in the future it's possible to use `Self` in the future.